components/BlocksTable/Table.tsx

```diff
@@ -89,13 +91,13 @@ const BlocksTable: FC<BlocksTableProps> = ({
       <Tbody>
         {data?.map(block => (
           <RowData
-            key={block.id}
+            key={block?.id}
```

Comment:
if block will be null it will produce key warning

Reply:
Yeah, I think a `block && <JSX/>` clause would be helpful.

Reply:
Fixed.